update.sh

```diff
@@ -1,4 +1,5 @@
-for repo in dummy-comp sr2014-comp sr2015-comp sr2016-comp ranker srcomp srcomp-http srcomp-scorer srcomp-screens srcomp-stream srcomp-cli srcomp-kiosk; do
+#!/bin/bash
```

Comment:
Does this strictly need `bash` or would `#!/bin/sh` suffice?

Reply:
`pushd`/`popd` are specifically bash builtins, so `#!/bin/bash` is required.